crypto/keys/README.md

```diff
@@ -0,0 +1,45 @@
+# Keys
+
+[![API Reference](https://godoc.org/github.com/cosmos/cosmos-sdk/crypto/keys?status.svg)](https://godoc.org/github.com/cosmos/cosmos-sdk/crypto/keys)
+
+# Keybase API
+
+## The Keybase interface
+
+The [Keybase](https://godoc.org/github.com/cosmos/cosmos-sdk/crypto/keys#Keybase) interface defines
+the methods that a type needs to implement to be used as key storage backend. This package provides
+few implementations out-of-the-box.
+
+## Constructors
+
+### New
+
+The [New](https://godoc.org/github.com/cosmos/cosmos-sdk/crypto/keys#New) constructor returns
+an on-disk implementation backed by LevelDB storage that has been the default implementation used by the SDK until v0.38.0.
+Due to [security concerns](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-006-secret-store-replacement.md), we recommend to drop
+it in favor of the `NewKeyring` or `NewKeyringFile` constructors. We strongly advise to migrate away from this function as **it may be removed in a future
+release**.
+
+### NewInMemory
+
+The [NewInMemory](https://godoc.org/github.com/cosmos/cosmos-sdk/crypto/keys#NewInMemory) constructor returns
+an implementation backed by an in-memory LevelDB storage that we've historically used for testing purposes or on-the-fly
```

Comment:
Is this true? I believed `MemDB` to be a true in-memory `map` db -- no LevelDB involved. I could be wrong here though.

Reply:
You're definitely right, just double checked. Will amend accordingly, thanks

Reply:
https://github.com/cosmos/cosmos-sdk/pull/5355/commits/6b3b8f3ae57089bfcd3dfc36d469bf412f10078e